aws/amazon-chime-sdk-js
# Add SignalingDrop counts meetingFailed event metric

**Issue #:**

**Description of changes:**
Add SignalingDropped counts meeting event metric. The SignalingDropped is now published as a meeting event in this recent  pr: #1580

**Testing**

1. Have you successfully run `npm run build:release` locally? yes
2. How did you test these changes? 
    1. Copy the code change and paste to CloudWatch dashboard for a preview. 
    2. Open the test app url deployed for testing and join a meeting
    3. Enable Network Link conditioner and select 100% loss. 
    4. The websocket connection will be closed
    5. Check the new signalingDropped metric in cloudwatch 
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? N/A
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? N/A
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
